native/package.json

```diff
@@ -11,20 +11,21 @@
     "android:build": "react-native run-android",
     "android": "npm run android:emulator & npm run android:build",
     "test": "jest --coverage",
-    "storybook": "storybook start -p 7007 --root ../"
+    "storybook": "storybook start -p 7007 --skip-packager | react-native start --watchFolders $PWD/.."
   },
   "rnpm": {
     "assets": [
       "../packages/bpk-svgs/dist/font/"
     ]
   },
   "devDependencies": {
-    "@storybook/react-native": "^3.3.10",
-    "babel-preset-react-native": "^4.0.0",
-    "react": "16.3.1",
-    "react-dom": "16.3.1",
-    "react-native": "0.55.3",
-    "react-test-renderer": "16.3.1",
+    "@storybook/react-native": "^3.4.11",
+    "babel-core": "7.0.0-bridge.0",
+    "metro-react-native-babel-preset": "^0.45.0",
+    "react": "16.5.1",
+    "react-dom": "16.5.1",
+    "react-native": "0.57.1",
+    "react-test-renderer": "16.5.1",
```

Comment:
* Updated Storybook to match the version in the root `package.json`. We should keep these in sync.
* Updated React as mandated in RN upgrade guide: https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#updating-to-this-version